source/renderer/app/components/widgets/NewsFeedIcon.tsx

```diff
@@ -1,33 +1,59 @@
-import React, { Component } from 'react';
+import React from 'react';
 import SVGInline from 'react-svg-inline';
 import classNames from 'classnames';
+import { PopOver } from 'react-polymorph/lib/components/PopOver';
+import { defineMessages, injectIntl } from 'react-intl';
 import newsFeedIcon from '../../assets/images/top-bar/news-feed-icon.inline.svg';
 import styles from './NewsFeedIcon.scss';
+import type { Intl } from '../../types/i18nTypes';
 
 type Props = {
+  intl: Intl;
   onNewsFeedIconClick: (...args: Array<any>) => any;
   newsFeedIconClass?: string;
   hasNotification: boolean;
   hasUpdate: boolean;
 };
-export default class NewsFeedIcon extends Component<Props> {
-  render() {
-    const {
-      onNewsFeedIconClick,
-      newsFeedIconClass,
-      hasNotification,
-      hasUpdate,
-    } = this.props;
-    const componentClasses = classNames([
-      styles.component,
-      hasNotification && !hasUpdate ? styles.notificationDot : null,
-      hasUpdate ? styles.updateDot : null,
-      newsFeedIconClass,
-    ]);
-    return (
-      <button className={componentClasses} onClick={onNewsFeedIconClick}>
-        <SVGInline className={styles.icon} svg={newsFeedIcon} />
-      </button>
-    );
-  }
+
+const messages = defineMessages({
+  iconTooltip: {
+    id: 'news.newsfeed.iconTooltip',
+    defaultMessage: '!!!Newsfeed',
+    description: 'Newsfeed',
+  },
+});
+
+function NewsFeedIcon({
+  intl,
+  onNewsFeedIconClick,
+  newsFeedIconClass,
+  hasNotification,
+  hasUpdate,
+}: Props) {
+  const buttonClasses = classNames([
+    styles.button,
+    hasNotification && !hasUpdate ? styles.notificationDot : null,
+    hasUpdate ? styles.updateDot : null,
+    newsFeedIconClass,
+  ]);
+  return (
+    <div className={styles.component}>
+      <PopOver
+        appendTo="parent"
+        delay={[300, 0]}
```

Comment:
I think we should move these magic numbers into a config file

Reply:
Updated, thanks!